dev/examples/Drag-drag-to-reorder.tsx

```diff
@@ -80,7 +74,7 @@ export const App = () => {
     // sibling.
     const moveItem = (i: number, dragOffset: number) => {
         const targetIndex = findIndex(i, dragOffset, positions)
-        if (targetIndex !== i) setColors(move(colors, i, targetIndex))
+        // if (targetIndex !== i) setColors(move(colors, i, targetIndex))
```

Comment:
Is this still a todo?

Reply:
I can just delete this file as there's a test